.github/workflows/cmake-format-check.yml

```diff
@@ -11,8 +11,9 @@ jobs:
     steps:
     - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
     - name: cmake-format lint action
-      uses: puneetmatharu/cmake-format-lint-action@efbb497b2a8badd2c9dc638faaf8ef4a9aa71bc8 # v1.0.4
+      uses: puneetmatharu/cmake-format-lint-action@2ac346e79e7ceac958bc637c1391285fb335ed7c # v1.0.5
       with:
         args: --config-files .cmake-format.py --in-place
+        file-regex: '(.*\.cmake$|.*\.cmake\.in$|CMakeLists.txt$)'
```

Comment:
```suggestion
        file-regex: '(\.cmake$|\.cmake\.in$|CMakeLists.txt$)'
```

Reply:
I feel like I will never understand regex. But I think we will have to match the whole filename not just the extension that was why I was using `.*`. I tested it here https://regex101.com/ but I might still be wrong :-D

Reply:
Then your 3rd alternative is wrong, the CMakeLists.txt

Reply:
Oh I take it back.  I expected whatever gets matched will have a relative path from the root of the repo